### DISCUSSION TITLE
Install instructions are using the wrong number for main menu

### DISCUSSION THREAD

**Discussion by turbo-david** [Nimdy/Dedicated_Valheim_Server_Script#215] [General]
The install instructions say to use option 4 to install the server on the latest build it is actually option 3.) instructions should be updated

**Comment by Nimdy** [comment]
![image](https://user-images.githubusercontent.com/16698453/112011502-9c2bc900-8b28-11eb-9231-c20806bb5b2e.png)

**Comment by Nimdy** [comment]
yeha I need to update the image. 

Thank you

**Comment by Nimdy** [comment]
updated! Thanks
